scripts/templates/webdriver.tpl.d.ts

```diff
@@ -391,6 +391,46 @@ declare namespace WebDriver {
         sessionId: string;
     }
 
+    // object with no match
+    interface ProtocolCommandResponse {
+        [key: string]: any;
+    }
+
+    // webdriver.json
+    interface SessionReturn {
+        sessionId: string,
+        capabilities: {
+            [key: string]: any
+        }
+    }
+
+    interface StatusReturn {
+        ready: boolean,
+        message: string,
+        [key: string]: any
+    }
+
+    interface WindowHandle {
+        handle: string,
+        type: string
+    }
+
+    interface RectReturn {
+        x: number,
+        y: number,
+        width: number,
+        height: number
+    }
+    // appium.json
+
+    interface StringsReturn {
+        [key:string]: string
+    }
+
+    interface SettingsReturn {
```

Comment:
SettingsReturn and ProtocolCommandResponse are the same, it might be better leave just one with some more abstract naming

Reply:
Changed `SettingsReturn`
If use `unknown`, we cannot access prop on it.